allure-support/src/main/kotlin/com/kaspersky/components/alluresupport/AllureSupportKaspressoBuilder.kt

```diff
@@ -74,7 +75,23 @@ fun Kaspresso.Builder.Companion.withForcedAllureSupport(
     forceAllureSupportFileProviders(instrumentalDependencyProvider)
     addRunListenersIfNeeded(instrumentalDependencyProvider)
 }.apply {
-    postInitAllure(shouldRecordVideo, builder = this)
+    postInitAllure(shouldRecordVideo, true, builder = this)
+}
+
+fun Kaspresso.Builder.Companion.withForcedAllureSupport(
```

Comment:
Please, remove the old builder. it bacame useless with your changes

Reply:
@devapro kindly reminder